1.ts

```diff
@@ -0,0 +1,12 @@
+@Component({
```

Comment:
Style guide: Strict Commit Messages: For bug fixes use fix prefix and the ticket identifier in the scope. example: 'fix(DDIAAS-8888): remove feature' - false